### PR TITLE
Release notes for Gluster 3.12.7

### PR DESCRIPTION
**Pull request by thotz**
Signed-off-by: Jiffin Tony Thottan <jthottan@redhat.com>

### CODE REVIEW

File: release-notes/3.12.7.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Release notes for Gluster 3.12.6
```
Comment (prashanthpai) [inline release-notes/3.12.7.md]: ` 3.12.7`

File: release-notes/3.12.7.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+This is a bugfix release. The release notes for [3.12.0](3.12.0.md), [3.12.1](3.12.1.md), [3.12.2](3.12.2.md), [3.12.3](3.12.3.md), [3.12.4](3.12.4.md), [3.12.5](3.12.5.md), [3.12.6](3.12.6.md) , [3.12.7](3.12.7.md)contain a listing of all the new features that were added and bugs fixed in the GlusterFS 3.12 stable release.
```
Comment (prashanthpai) [inline release-notes/3.12.7.md]: I don't think you need to add a link to this document i,e ` 3.12.7`

Comment (thotz) [reply release-notes/3.12.7.md]: usually I follow the existing format, I can remove it if needed

File: release-notes/3.12.7.md (hunk 1 above)
Comment (thotz) [reply release-notes/3.12.7.md]: will do that

**Pull request closed without merging**